<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>evaluator</class>
 <widget class="QWidget" name="evaluator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>INS-GPS Evaluation</string>
  </property>
  <widget class="QLineEdit" name="result_folder_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>140</y>
     <width>351</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="evaluate_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>140</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>EVALUATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="data_folder_select_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Data folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="result_folder_select_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Results folder</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="attitude_group">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>70</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Attitude</string>
   </property>
   <widget class="QCheckBox" name="pitch_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pitch</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="roll_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="navigation_group">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>90</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Navigation</string>
   </property>
   <widget class="QCheckBox" name="course_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Course</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="velocity_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Velocity</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="position_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="data_folder_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>351</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="extra_group">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>520</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Additional parameters</string>
   </property>
   <widget class="QCheckBox" name="sat_num_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Active SRNS receivers</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sphere_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Position on Earth</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sat_mean_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SRNS Averaging</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pos_ratio_checkbox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Position ratios</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="vel_ratio_checkbox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Velocity ratios</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="course_ratio_checkbox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Course ratios</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="show_button">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>20</y>
     <width>140</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SHOW POINTS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
